com.ibm.wala.cast.python.test/data/tf2_test_callbacks.py

```diff
@@ -0,0 +1,12 @@
+import tensorflow as tf
+
+def get_dataset(input_context):
```

Comment:
Which function is getting "called back?"

Reply:
`get_dataset`

Reply:
It is added